dap_job_quality/utils/jobbert.py

```diff
@@ -0,0 +1,74 @@
+import time
```

Comment:
This blows my mind! I thought that you could only use `SentenceTransformers` with a sentence model (rather than a masking / classification etc one) 🤯 That's a pretty big time saving! Thanks Liz